content/changelog/2024-02-09-console.md

```diff
@@ -0,0 +1,67 @@
+
+If you installed this extension previously and want to upgrade to the latest version, please refer to [Update an extension version](/docs/extensions/pg-extensions#update-an-extension-version) for instructions.
+
+For a complete list of Postgres extensions supported by Neon, see [Postgres extensions](/docs/extensions/pg-extensions).
+
+### IPv6 support coming soon
+
+Neon will soon start accepting IPv6 connections at no extra charge. We will keep supporting IPv4 connections, also at no extra charge.
+
+We'll progressively roll out IPv6 to all our supported regions during February. You shouldn’t experience any downtime or disturbances: if your host is IPv6 enabled and its client library prefers IPv6 by default, your new connections will start using it. No action is required from you. 
+
+If you have any questions, reach out to us in [Discord](https://discord.gg/92vNTzKDGp).
+
+### Support for prepared statements with PgBouncer
+
+The PgBouncer version used by Neon to offer pooled connection support was updated to [version 1.22.0](https://www.pgbouncer.org/changelog.html#pgbouncer-122x). This version of PgBouncer adds support for `DEALLOCATE ALL` and `DISCARD ALL` in `transaction` mode. In Neon, the new version of PgBouncer enables support for protocol-level prepared statements.
+
+Support for protocol-level prepared statements with PgBouncer means you can now use prepared statements with pooled connections. Using prepared statements can help boost query performance while providing an added layer of protection against potential SQL injection attacks. To learn more, see [Optimize queries with PgBouncer and prepared statements](/docs/connect/connection-pooling#optimize-queries-with-pgbouncer-and-prepared-statements) in the docs or read our [blog post](tbd).
+
+### Logical replication enhancements
+
+- `CREATE PUBLICATION my_publication FOR ALL TABLES` syntax is now supported in Neon. Previously, `FOR ALL TABLES` was not supported, and multiple tables could only be specified using `CREATE PUBLICATION my_pub FOR TABLE <table1>, <table2>` syntax.
+- The [neon_superuser](/docs/manage/roles#the-neonsuperuser-role) role is now a member of the Postgres `pg_create_subscription` role, which allows users with `CREATE` permission on a database to issue `CREATE SUBSCRIPTION`. The `pg_create_subscription` role is only available as of Postgres 16. The `neon_superuser` role in Postgres 14 and 15 can issue `CREATE SUBSCRIPTION` with only `CREATE` permission on the database. This change enables defining a Neon database as a logical replication subscriber. Previously, a Neon database could only be defined as a logical replication publisher.
```

Comment:
let's not announce it yet. Because it means that users can create logical replication subscriptions